postgresql-11.5/src/include/fe_utils/printbteq.h

```diff
@@ -150,6 +151,7 @@ typedef struct printTableContentBteq
     const char **header;        /* Pointer to the last added header */
     const char **cells;            /* NULL-terminated array of cell content
                                  * strings */
+    int           table_width;
```

Comment:
Why do we need this redefined in different structures? Which one do we use?

Reply:
first we give the value printQueryOptBteq topt.table_width, and then we give value the content-> table_width = opt-> table_width in the printTableInitbteq function, then we call the printTablebteq function with the printTableContentBteq cont arguments.So we use the variables of the two structures